gitpod-io/website
# Revamped `/dedicated` page & sales form

## Description

This PR introduces:
- revamped `/dedicated` page
- New subject for `Dedicated self-serve` in sales (`/contact/sales`) form

## Related Issue(s)

Fixes MKWEB-71

## How to test



- Go to [`/dedicated`](https://deploy-preview-3804--gitpod-kumquat.netlify.app/dedicated) & see 👀

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing



## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.


  </details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3804"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
